tests/System.IO.Pipelines.Tests/PipeTest.cs

```diff
@@ -9,16 +10,19 @@ public abstract class PipeTest : IDisposable
     {
         protected const int MaximumSizeHigh = 65;
 
+        private readonly TestMemoryPool _pool;
+
         protected Pipe Pipe;
-        private readonly MemoryPool _pool;
 
-        protected PipeTest()
+        protected PipeTest(int pauseWriterThreshold = 65, int resumeWriterThreshold = 6)
```

Comment:
nit: Use MaximumSizeHigh?

Reply:
why?

Reply:
We have a constant already. I assume this 65 has the same semantic meaning. If not, can you add a comment why these constant values are special?

Reply:
Ah, my bad, you are completely right.